ios/Podfile

```diff
@@ -2,7 +2,7 @@ require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
 
-platform :ios, '10.0'
+platform :ios, '11.0'
```

Comment:
Intended to update this?

Reply:
yeah, the expo splash library required it.